metadata/src/test/java/org/apache/kafka/metadata/util/MetadataFeatureUtil.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.metadata.util;
+
+import org.apache.kafka.server.common.MetadataVersion;
+import org.mockito.internal.util.MockUtil;
+
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+public class MetadataFeatureUtil {
```

Comment:
I've been thinking about this more, and I think maybe we should just claim a metadata version number. It's time.

Maybe even move KIP-966 up a notch since we may finish before they do. I guess let's discuss offline.

Reply:
I agree, it is getting a bit annoying to have to mock MetadataVersion everywhere.

Reply:
I'm going to raise a separate PR to handle this. But I just wanted to comment here for the sake of completeness :)

Reply:
Thank you! 👍

Reply:
There's a JIRA for that already https://issues.apache.org/jira/browse/KAFKA-15893